### ISSUE TITLE
Sign In issue, or perhaps just confused

### ISSUE THREAD

**Issue by AXLrYRr2WrFt2w** [TrsstProject/trsst#15]
I used "New Account" to create an account on my laptop machine.  The next day I was hoping to use trsst from my desktop machine.  I downloaded the app and installed.  When I click on 'Sign In' no dialog appears asking for a id/password.  Clicking New Account does bring up a dialog box for new account creation.

Version (0.2) alpha  Mac OS X 10.9.2 Build 13C64

**Comment by mmpowers** [comment]
Yes, this is by design -- it's our whole "you keep your keys" thing.  

When you create your account, your client app is generating a keypair and storing it in a PKCS12 file encrypted with your passphrase.  We don't keep this file on our servers.  If you'd like to use the same account from multiple computers, just copy this file (in the "Trsst Accounts" folder in your home directory, ending in .p12) into the same location on the other computer.

The confusion you're experiencing is very common, and this is shaping up as one of our major usability issues.  It may be that we allow people host their own keys in the cloud, as long as they acknowledge that they leave themselves at greater risk.

**Comment by AXLrYRr2WrFt2w** [comment]
Thanks.  I thought it was likely my confusion of how to keep the machines tied together.  

Will transfer the file soon when I'm near the laptop again and try it out.

In terms of the usability, perhaps some sort of feedback if user click "Sign In" and the accounts folder is empty.
